### ISSUE TITLE
SSO Project AAD B2C technical issues resolution

### ISSUE THREAD

**Issue by jon-mc-git** [bcgov/cloud-pathfinder#1196]
**Describe the issue**
To help resolve technical issues involving the SSO teams stand up of the Azure AD B2C service for usage testing 

**Additional context**
In order to stand up the Azure AD B2C service it was found that Global Admin is needed.  This may possibly be done by another role or roles but it is unknown if this is indeed possible, and\or if these other roles could be used to administer the service on a more daily basis.
- If this on-going management can be restricted to a B2C 'instance'
- If management of this B2C service is more global in nature, and would involve the SSO project team to manage this for any other B2C instances that may have to be created in the future (ie. future responsibility accepted and security practices established that would safe guard this higher level management process - ie. An on-going Operations concern)

**Definition of done**
- Coordinate a solution between MS best practices and SSO team implementation
- Conduct research where necessary here to bring a solution together for the SSO testing team
- Discuss any on-going operational concerns to be aware for for SSO team, so will be part of their findings
- Ensure Azure AD B2C service is enabled and coordinate between the SSO testing team and the ADMS group when needed

**Comment by jon-mc-git** [comment]
B2C guest tenancy will be place under an alternate Dev tenancy being created with the ADMS group.  If there is any technical issue within this upcoming implementation then a PoC B2C guest tenancy will be created under TIDIR - Upcoming work with ADMS group will move this B2C PoC environment creation request forward

**Comment by ActionAnalytics** [comment]
This might be solved via the pCard approach Julian is doing if this gets blocked by the time to complete a full review from ADMS

**Comment by jon-mc-git** [comment]
This was solved by way of pcard for the SSO team, as the ADMS team are still working on resolving whether there would be any possible connection back to our BC Gov prod tenancy (IDIR based).  Once they resolve it, a tenancy can be created for the Azure SEA Test enviro - likely will be based in TIDIR

**Comment by jon-mc-git** [comment]
Once ADMS group is comfortable with this process the SSO group can move to creating a 'B2C Service' (tenancy) for more permanent testing under a TIDIR Azure tenancy.  New ticket will be created once SSO team identifies that they are ready to move forward - Closing this initial issue
